packages/driver/cypress/e2e/commands/actions/type.cy.js

```diff
@@ -360,7 +360,25 @@ describe('src/cy/commands/actions/type - #type', () => {
       cy.get(':text:first').type('foo', { animationDistanceThreshold: 1000 }).then(() => {
         const { args } = cy.ensureElementIsNotAnimating.firstCall
 
-        expect(args[1]).to.deep.eq([fromElWindow, fromElWindow])
+        const plusMinusOne = (n) => [n - 1, n + 1]
+
```

Comment:
Had some weird flake, CI and firefox only - things were off by 1px for some reason, so I added this.

Will make a JIRA ticket to follow up and see if we can identify the source of the flake.